### DISCUSSION TITLE
Connection to car lost.

### DISCUSSION THREAD

**Discussion by user7590** [TobiasLaross/Hass-Lynk-Co#40] [Q&A] [answered]
After a few days fiddling around with this wonderful component, suddenly the connection to my car is lost. Also in the official app: the car doesnt show up anymore. Any thoughts?
Thank you

**Comment by TobiasLaross** [comment, marked as answer]
I have had similar experience a while back, I had 10min scan_interval but it might have been lynk servers that was unstable because I noticed other app users had issues as well. 
what refresh rate did you have?

I disconnected the integration for 24h and then it started working again but I have no idea why it happens unfortunately

**Comment by user7590** [comment]
Strange yes, it's working again now. Unstable i think yes. Thanks Tobias.
